third_party/3/pkg_resources/__init__.pyi

```diff
@@ -5,6 +5,7 @@ from abc import ABCMeta
 import importlib.abc
 import types
 import zipimport
+import packaging.markers
```

Comment:
The packaging module does not exist yet in typeshed. (It also seems not to exist at runtime, but is imported using some trickery in `pkg_resources`. The easiest solution is to mark `marker` as `Optional[Any]` for now.

Reply:
Thanks for the suggestion, I was a little lost how to tackle this.